### PR TITLE
Feature/fe-be-signin

### PR DESCRIPTION
**Pull request by oyakovytskyi**
## Describe your changes

- Connect BE to FE login 

## Issue ticket code (and/or) and link

- [Link to JIRA ticket](https://codecraftersintership.atlassian.net/jira/software/projects/SCRUM/boards/1?selectedIssue=SCRUM-4)

### **General**

- [x] Assigned myself to the PR
- [x] Assigned the appropriate labels to the PR
- [x] Assigned the appropriate reviewers to the PR
- [ ] Updated the documentation
- [x] Performed a self-review of my code
- [x] Types for input and output parameters
- [x] Don't have "any" on my code
- [x] Used the try/catch pattern for error handling
- [x] Don't have magic numbers
- [x] Compare only with constants not with strings
- [x] No ternary operator inside the ternary operator
- [x] Don't have commented code
- [x] no links in the code, env links should be in env file (for example: server url), constant links (for example default avatar URL) should be in constant file.
- [x] Used camelCase for variables and functions
- [x] Date and time formats are on the constants
- [x] Functions are public only if it's used outside the class
- [x] No hardcoded values
- [ ] covered by tests
- [x] Check your commit messages meet the [conventional commit format](https://www.conventionalcommits.org/en/v1.0.0/).

### Frontend

- [x] Components and business logic are separated
- [x] Colors, Font Size, and Font Name is on the theme or in the constants
- [x] No text in the components, use i18n approach
- [x] No inline styles
- [x] Imports are absolute
- [ ] Attach a screenshot if PR has visual changes.

### CODE REVIEW

File: src/pages/SignIn/useSignIn.ts [hunk 1]
```diff
@@ -1,6 +1,11 @@
-import React from 'react';
-import { useState } from 'react';
+// src/pages/SignIn/useSignIn.ts
```
Comment (lyudad) [inline src/pages/SignIn/useSignIn.ts]: delete if not needed

File: src/pages/SignIn/useSignIn.ts [hunk 2]
```diff
@@ -11,23 +16,61 @@ interface UseSignInReturnType {
...
+
+    try {
+      const response = await axios.get(`/auth/${email}`, {
+        params: { invitationToken },
+      });
+
+      const { token } = response.data;
+
+      if (token) {
+        localStorage.setItem('accessToken', token);
```
Comment (lyudad) [inline src/pages/SignIn/useSignIn.ts]: better use redux-persist library

File: src/store/store.ts [hunk 3]
```diff
@@ -1,12 +1,20 @@
 import { configureStore } from '@reduxjs/toolkit';
 
-import exampleReducer from '@/store/exampleSlice';
+import exampleReducer from '@/store/slices/exampleSlice';
+import authReducer, { setAccessToken } from '@/store/slices/authSlice';
+
+const loadAccessToken = (): string => {
+  return localStorage.getItem('accessToken') || '';
```
Comment (lyudad) [inline src/store/store.ts]: use redux-persist and this line will not be needed it have local storage underhood

File: src/pages/SignIn/useSignIn.ts (hunk 1 above)
Comment (oyakovytskyi) [reply src/pages/SignIn/useSignIn.ts]: done

File: src/pages/SignIn/useSignIn.ts (hunk 2 above)
Comment (oyakovytskyi) [reply src/pages/SignIn/useSignIn.ts]: ok, will try

File: src/store/store.ts (hunk 3 above)
Comment (oyakovytskyi) [reply src/store/store.ts]: removed

File: .env
Comment (lyudad) [inline .env:1]: что это такое? 1.1.9

Comment (oyakovytskyi) [reply .env:1]: removed

**Review by lyudad — APPROVED** [review]

**Pull request merged**